README.md

```diff
@@ -1,16 +1,15 @@
 # Miami landing page
 In this task, you will learn how to implement a landing page. To do that:
 
+Replace `<your_account>` with your Github username and copy the links to Pull Request description:
+[DEMO LINK](https://Oleksandr-Naumchak/git.github.io/layout_miami/)
```

Comment:
Link doesn't work, did you deploy your work?

Reply:
Fix link on correct - https://oleksandr-naumchak.github.io/layout_miami/